R/ggplotly.R

```diff
@@ -806,7 +806,12 @@ gg2list <- function(p, width = NULL, height = NULL,
       
       # do some stuff that should be done once for the entire plot
       if (i == 1) {
-        axisTickText <- longest_element(axisObj$ticktext)
+        # Split ticktext elements by "\n"  to account for linebreaks
+        axisTickText <- strsplit(as.character(axisObj$ticktext), split = "\n")
+        # Get longest element for each tick
+        axisTickText <- lapply(axisTickText, longest_element)
+        # Longest element of all ticks
+        axisTickText <- longest_element(axisTickText)
```

Comment:
I think this would be slightly faster

```suggestion
        axisTickText <- longest_element(unlist(axisTickText))
```